automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Modal.scala

```diff
@@ -77,5 +77,15 @@ class SynchronizeMethodAccessModal(id: String)(implicit webDriver: WebDriver) ex
 
 }
 
+class PreviewModal(id: String)(implicit webDriver: WebDriver) extends OKCancelModal(id) {
+  def getBucket = testId("Google Bucket").webElement.findElement(By.xpath("..")).getText()
+  def getObject = testId("Object").webElement.findElement(By.xpath("..")).getText()
```

Comment:
alternative to `..` is `"//*[@data-test-id='Google Bucket']/parent::node()"`. both works, one is shorter.